src/App.js

```diff
@@ -45,6 +58,7 @@ class App extends Component {
       <main className="main-container">
         <h1>Rotten Tomatillos</h1>
         <h2 className="opening-line">Where your imagination comes to life on the big screen</h2>
+        {this.state.error ? <ErrorModal message={this.state.error}/> : null}
```

Comment:
Very nice!! Great job getting this tricky conditional rendering set up. One thing you might want to change though is to move this out into a function. It can be pretty much the same code, but will live outside of your render function. Then, you can call it in render. Its a good way to keep your code clean!